### ISSUE TITLE
JTableHeaderLocation.columnName() doesn't work for reordered columns

### ISSUE THREAD

**Issue by jedwards1211** [alexruiz/fest-swing-1.x#17]
Hi Alex, this is my first contribution to FEST, woohoo :)

Here's the code I'm referring to (in https://github.com/alexruiz/fest-swing-1.x/blob/master/fest-swing/src/main/java/org/fest/swing/driver/JTableHeaderLocation.java):

``` java
  @RunsInCurrentThread
  private String columnName(@Nonnull JTableHeader tableHeader, int index) {
    return tableHeader.getTable().getModel().getColumnName(index);
  }
```

columnName(2) would return the name of the column at **model index** 2.  Correct me if I'm wrong, but it should be returning the name of the column at **view index** 2, which may be different if the view columns have been reordered.

This should fix it:

``` java
  @RunsInCurrentThread
  private String columnName(@Nonnull JTableHeader tableHeader, int index) {
    return tableHeader.getTable().getColumnName(index);
  }
```

**Comment by manasouza** [comment]
Good point! I already been annoyed by this model/view JTable concepts....guess must be monitored cases when we have to apply convertColumnIndexToModel(int) and convertColumnIndexToView(int)....maybe we'll need two methods, one for each case...

**Comment by jedwards1211** [comment]
Well in this case, it _must_ use the view index, or it may click a column
with a different name...
